src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/Instance.sql

```diff
@@ -23,20 +23,22 @@ CREATE TABLE dbo.Instance (
     HasFrameMetadata        BIT                        NOT NULL DEFAULT 0
 ) WITH (DATA_COMPRESSION = PAGE)
 
+-- Primary index, also used in views
 CREATE UNIQUE CLUSTERED INDEX IXC_Instance on dbo.Instance
 (
+    PartitionKey,
+    StudyKey,
     SeriesKey,
     InstanceKey
 )
 
---Filter indexes
--- Used in AddInstance, DeleteInstance, DeleteDeletedInstance, QIDO
-CREATE UNIQUE NONCLUSTERED INDEX IX_Instance_StudyInstanceUid_SeriesInstanceUid_SopInstanceUid_PartitionKey on dbo.Instance
+-- Used in AddInstance, DeleteInstance
+CREATE UNIQUE NONCLUSTERED INDEX IX_Instance_PartitionKey_StudyInstanceUid_SeriesInstanceUid_SopInstanceUid on dbo.Instance
```

Comment:
Isn't this used in QIDO instancewithinastudyandseries? The where condition we do is 
```
WHERE 1 = 1
      AND st.PartitionKey = 1
      AND i.SopInstanceUid=''
      AND st.StudyInstanceUid=''
      AND se.SeriesInstanceUid=''
```

But that could be all from instance table like below
```
WHERE 1 = 1
      AND i.PartitionKey = 1
      AND i.SopInstanceUid=''
      AND i.StudyInstanceUid=''
      AND i.SeriesInstanceUid=''
```

This will reduce the joins and use the index